.github/workflows/switches.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: switches
+
+on:
+  pull_request:
+    paths:
+      - 'switch-configurations/**'
+  push:   # This is only run when PRs are merged into master
+    branches:
+      - master
+  workflow_dispatch:
+
+
+jobs:
+  linter:
+    name: perl_lint
+    runs-on: ubuntu-latest
+    container:
+      # Perl 5 latest: 12-30-21
+      image: perl@sha256:1ec7df9b6c70e31b05d937f929ddfe06a6a1966d98c09ae78ecfbb391a364739
+    steps:
+      - name: checkout
+        id: checkout
+        uses: actions/checkout@v2
+      - name: lint
+        working_directory: switch-configuration
+        run: make lint
```

Comment:
I like the switch to `Makefile` for this. I think we should also ticket out doing this in the Ansible section.

Reply:
I also support the switch to Makefile.

Reply:
glad you both approve! :+1: